ablator/config/rclone.py

```diff
@@ -0,0 +1,32 @@
+from ablator.config.main import configclass
+from ablator.config.types import Stateless
+from ablator.modules.storage.rclone import RcloneConfig
+
+
+@configclass
+class GcsRcloneConfig(RcloneConfig):
```

Comment:
Is it possible to use a generic RcloneConfig that will address all supported data storage options?

Reply:
There is no generic configs for all the remote options, if you want to do so, I can add all the possible fields into one config, but I think it is confusing for users